src/core/server/rendering/views/template.tsx

```diff
@@ -120,7 +148,7 @@ export const Template: FunctionComponent<Props> = ({
           data-test-subj="osdLoadingMessage"
         >
           <div className="osdLoaderWrap">
-            {openSearchLogo}
+            {openSearchLogoSpinner}
```

Comment:
Let's remove `opensearchLogo` if not needed.

Reply:
It's still needed - it's referenced further down on a second use case where it's not a loading spinner but rather a static image with an error message.